### PR TITLE
Symorton/hashed pin updates

### PR DESCRIPTION
**Pull request by symorton**
**Problem:**
There is new logic for issuance pin.

**Solution:**
Hashed the pin and added it to top level object.

**Validation:**
Create Unit Tests

**Type of change:**
- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [X] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [X] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: VCEntities/VCEntities/issuance/IssuanceResponseContainer.swift [hunk 1]
```diff
@@ -11,26 +11,21 @@ public struct IssuanceResponseContainer: ResponseContaining {
...
+    public var issuancePin: IssuancePin? = nil
     public var issuanceIdToken: String? = nil
     public var requestedIdTokenMap: RequestedIdTokenMap = [:]
     public var requestedSelfAttestedClaimMap: RequestedSelfAttestedClaimMap = [:]
     public var requestVCMap: RequestedVerifiableCredentialMap = [:]
     
     public init(from contract: Contract,
                 contractUri: String,
-                expiryInSeconds exp: Int = 3000,
-                rawIssuerIdToken: String? = nil,
```
Comment (nithyaganeshng) [inline VCEntities/VCEntities/issuance/IssuanceResponseContainer.swift:23]: There is no change in idtoken being sent. Was this change intentional?

Comment (symorton) [reply VCEntities/VCEntities/issuance/IssuanceResponseContainer.swift:23]: yep, its an optimization that I saw and wanted to fix

**Review by nithyaganeshng — APPROVED** [review]
